src/Common/src/Interop/Windows/advapi32/Interop.ClaimSecurityAttributes.cs

```diff
@@ -0,0 +1,128 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Runtime.InteropServices;
+
+internal partial class Interop
+{
+    [StructLayout(LayoutKind.Explicit)]
+    internal struct CLAIM_SECURITY_ATTRIBUTE_INFORMATION_V1
+    {
+        // defined as union in CLAIM_SECURITY_ATTRIBUTES_INFORMATION
+        [FieldOffset(0)]
```

Comment:
It's a bit strange seeing an explicit layout like this with a single field.

Reply:
I assume it was done that way (in netfx) to be prepared if Windows ever expanded their union type beyond one field

https://msdn.microsoft.com/en-us/library/windows/desktop/hh448481(v=vs.85).aspx